test/yup-middleware.spec.ts

```diff
@@ -127,6 +127,52 @@ describe('express yup middleware', () => {
     })
   })
 
+  describe('using yup validation options', () => {
+    const schemaValidator: ExpressYupMiddlewareInterface = {
+      schema: {
+        body: {
+          yupSchema: Yup.object().shape({
+            firstTestBodyProperty: Yup.string().required('requiredFirstTestBodyProperty'),
+            secondTestBodyProperty: Yup.string().required('requiredSecondTestBodyProperty'),
+          }),
+          validateOptions: {
+            abortEarly: false,
+          },
+        },
+      },
+    }
+
+    beforeEach(() => {
+      const app = createAppWithPath({
+        path: '/test',
+        middleware: expressYupMiddleware({ schemaValidator }),
+      })
+
+      agent = request(app)
+    })
+
+    it('returns an array with all the errors when validating multiple error messages on the same property', async () => {
```

Comment:
I didn't understand what you meant with on the same property, once you have two different properties being validated: firstTestBodyProperty and secondTestBodyProperty

Reply:
I've changed the text. It's more specific now, pointing to the `body` property.